server/chat.ts

```diff
@@ -1703,6 +1703,21 @@ export const Chat = new class {
 		if (!htmlContent) return '';
 		return htmlContent.replace(/<[^>]*>/g, '');
 	}
+	/**
+	 * Validates input regex and ensures it won't crash.
+	 */
+	validateRegex(word: string) {
```

Comment:
I wonder if we want to also check if the regex starts/ends with `|`, because that's definitely a mistake.

Reply:
Seems useful. Did it.